rules/do-you-use-ai-for-ui/rule.md

```diff
@@ -0,0 +1,45 @@
+---
+type: rule
+archivedreason: 
+title: Do You Use AI in UI Design?
+guid: b6849044-e11e-4a25-95a1-457493ee9eab
+uri: ai-in-ui-design
+created: 2024-01-09T19:11:41.0000000Z
+authors:
+- title: Luke Cook
+  url: https://ssw.com.au/people/luke-cook
+- title: Gordon Beeming
+  url: https://ssw.com.au/people/gordon-beeming
+- title: Joseph Fernandez
+  url: https://ssw.com.au/people/joseph-fernandez
+- title: William Liebenberg
+  url: https://ssw.com.au/people/william-liebenberg
+related: []
+redirects: []
+
+---
+
+
+When a web developer is handed design images, it can be a drag to spend the next few hours (or days/weeks) building the baseline components, HTML, and CSS before getting to the fun stuff.
+
+Thankfully, AI-assisted tools are beginning to help with the more mundane aspects of this process, and sites such as [v0.dev](https://v0.dev) can produce React components you can drop into your project to give you a huge headstart on the work!
+
+`youtube: https://www.youtube.com/watch?v=By9wCB9IZp0`
+
+## Boilerplate
+
+Upload your designers' images and tweak the output with a few prompts to get ready-made components for your app. Just be careful - these tools will often take "creative liberties" with the designs, so keep an eye out.
+
+## DIY devs
+
+If you're a dev that likes (or is forced) to moonlight as a designer these tools can be a great timesaver in creating professional looking web components in a short amount of time. These tools aren't a substitute for actual designers, but if your choice as a developer is between these tools or MS Paint, then you'll get a lot of value from them.
+
+## PoCs and Inspiration
+
+Generative UI tools can be a great way to collaborate with stakeholders in realtime, as you can generate ideas and iterate over them using natural language prompts. This can give developers or designers a useful starting point and frame of reference to build a highly polished design.
+
+### Tools
+
+[v0.dev](https://v0.dev)
```

Comment:
Hey @lukecookssw 

- [ ] I would make this a list. Please action if agreed ;)

Reply:
Done